src/TypeSpec.Extension/Emitter.Csharp/src/lib/converter.ts

```diff
@@ -316,7 +384,8 @@ function fromBytesType(bytesType: SdkBuiltInType): InputPrimitiveType {
 
 function fromStringType(
     program: Program,
-    stringType: SdkType
+    stringType: SdkType,
+    raw?: Type // we need the extra raw here because the format decorator is added to the property/parameter, but the raw in stringType is the type itself, and it does not have the format decorator we want.
```

Comment:
What the raw? Is it `ModelProperty`? and it is a item related with `stringType`? If we set a parameter which need special relationship with another one, it may not good because customer may pass a wrong raw.